keps/sig-network/0752-endpointslices/README.md

```diff
@@ -304,6 +301,94 @@ following well known key:
 If the Kubernetes Service has topological keys specified, the corresponding node
 labels will be copied to endpoint topology.
 
+### Converting Topology in EndpointSlice GA
+
+For beta resources that have set the topology field, the string to string map
+will be converted into an annotation on the v1 resource.
+
+The size limits of the topology field is 63B per key and per value, with a
+maximum of 126B per key-value pair. The maximum number of endpoints in an
+EndpointSlice is 1000.
+
+```
+{Max Key-Value Pairs} * ({Max Key Size} + {Max Value Size}) * {Max Endpoints}
+16 * (63 + 63) * 1000 = 2,016,000b
+```
+
+The maximum value of the combined maps in the endpoints is almost a factor of
+10 greater than the max annotation size (256kB). To reduce the size of the
+annotation, only the first 100 endpoints will be preserved per EndpointSlice
+and each endpoint's map will be base64 encoded.
+
+The annotation key will be **topology.kubernetes.io** and the value will be a
```

Comment:
Nit: I think I'd recommend using something EndpointSlice specific like `endpointslice.kubernetes.io/topology` instead. I think we may want to reserve something like `topology.kubernetes.io` for something more generic in the future.

Reply:
`endpointslice.kubernetes.io/v1beta1-topology` ?

Reply:
Yeah that's even better, something that we're unlikely to want to use for any other purpose in the future.